src/Microsoft.ML/Microsoft.ML.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MlNetMklDeps" Version="0.0.0.1" />
```

Comment:
>  <PackageReference Include="MlNetMklDeps" Version="0.0.0.1" /> [](start = 2, length = 63)

note about having to split this as a separate project, based on @eerhardt feedback on PR #594

Reply:
Correct.  We shouldn't be referencing `MlNetMklDeps` from the `Microsoft.ML` nuget package.